### PR TITLE
653 Server Status endpoint

### PR DESCRIPTION
**Pull request by summer-cook**
# Story
Sets up the server status endpoint at `/api/sushi/r51/status`

# Related
- #653 

# Expected Behavior Before Changes
- the `/api/sushi/r51/status` endpoint just had a simple message that said status: ok

# Expected Behavior After Changes
- the `/api/sushi/r51/status` endpoint returns a request body array that fits the counter structure

# Screenshots / Video

<details>
<summary>JSON response body</summary>
image
</details>

### CODE REVIEW

**Review by labradford — APPROVED** [review]

**Review by alishaevn — APPROVED** [review]
approving assuming you'll get the pipeline green before merging.

**Pull request merged**
